Dockerfile

```diff
@@ -54,7 +54,7 @@ ARG SOCI_SNAPSHOTTER_VERSION=0.6.1
 FROM --platform=$BUILDPLATFORM tonistiigi/xx:1.4.0 AS xx
 
 
-FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-bullseye AS build-base-debian
+FROM --platform=$BUILDPLATFORM golang:$GO_VERSION${GO_VERSION:+-}bullseye AS build-base-debian
```

Comment:
?

Reply:
Right now, go is pinned (^) to the latest released RC, but in normal times, we might want to just be on `golang:bullseye` (or `golang:alpine`). With the current Dockerfile, this is not possible.
This change makes it so that an empty `GO_VERSION` will give us `golang:bullseye`, while a non-empty one will give us `golang:foo-bullseye`.

Of course if we do your suggestion above we will always set the go version to the latest available (detecting from https://go.dev/dl/?mode=json&include=all ) and this is less useful, so, I can remove that.